Src/StartMenu/StartMenuDLL/MenuContainer.cpp

```diff
@@ -5002,7 +5002,7 @@ void CMenuContainer::SetSearchState( TSearchState state )
 	if (m_Submenu!=-1)
 	{
 		InvalidateRect(&m_Items[m_Submenu].itemRect);
- 		SetSubmenu(-1);
```

Comment:
Looks like unintended change.

Reply:
That didn't actually change, Github just marked it as changed for some odd reason

Reply:
This used spaces and a tab, I've changed it to tabs

Reply:
Please, don't do unrelated changes.
This line is not related to your PR changes, so no need to touch it.